irohad/torii/processor/transaction_processor.hpp

```diff
@@ -21,24 +21,28 @@
 #include <rxcpp/rx.hpp>
 #include <model/transaction.hpp>
 #include <model/client.hpp>
+#include <validation/stateless/validator.hpp>
 
 namespace iroha {
   namespace torii {
 
     /**
-     * Transaction processor is interface with start point
+     * Transaction handler is interface with start point
```

Comment:
Changing names like this should really be approved by me, as this is a core component of the system.

Generally I think TransactionHandler is an okay name, but everyone needs to know why the name changed.